src/components/learner-credit-management/AssignmentTableCancel.jsx

```diff
@@ -17,9 +21,21 @@ const calculateTotalToCancel = ({
   return assignmentUuids.length;
 };
 
-const AssignmentTableCancelAction = ({ selectedFlatRows, isEntireTableSelected, tableInstance }) => {
-  const assignmentUuids = selectedFlatRows.map(row => row.id);
-  const assignmentConfigurationUuid = selectedFlatRows[0].original.assignmentConfiguration;
+const AssignmentTableCancelAction = ({
+  selectedFlatRows, isEntireTableSelected, learnerStateCounts, tableInstance, enterpriseId,
+}) => {
+  const { subsidyAccessPolicyId } = useBudgetId();
+  const { data: subsidyAccessPolicy } = useSubsidyAccessPolicy(subsidyAccessPolicyId);
+  const { assignmentConfiguration } = subsidyAccessPolicy;
```

Comment:
Updates how assignment configuration is retrieved or cancel/remind all. Not dependent on the first value of the array having the assignment configuration. Changed for edge case during remind all where none of the filtered selectedRows are remindable, resulting in an empty array.